proto/beacon/p2p/v1/types.proto

```diff
@@ -170,14 +170,23 @@ message BeaconBlock {
   // repeated AttestationRecord attestations = 6;
   // Specials consist of exist, penalities, etc.
   repeated SpecialRecord specials = 7  [deprecated=true]; // Deprecated in favor of unify specials object w/ attestations.
-  repeated bytes proposer_signature = 8; // Type of [uint384]?
-
+  repeated bytes signature = 8; // Type of [uint384]?
```

Comment:
Did we define a uint384 type in proto in a separate file before? Can we use that

Reply:
We should. Similar to what we did with `Uint32List`